docs/adr/adr-005-record-architecture-decisions.md

```diff
@@ -0,0 +1,21 @@
+# 1. Record architecture decisions
+
+Date: 2020-10-21
+
+## Context
+
+While we have already recorded architectural decisions for this project, the structure of these documents has been arbitrary, since there was no guidance to follow. Recently we found that creating a new ADR was [hindered](https://github.com/alphagov/email-alert-api/pull/1441#discussion_r508729384) by wanting to be consistent with the structure of previous ADRs.
```

Comment:
The structure wasn't arbitrary it was based on: https://github.com/joelparkerhenderson/architecture_decision_record/blob/master/adr_template_for_alexandrian_pattern.md

Reply:
Oh fair enough. It was an attempt to say there was no documented reasoning behind the layout - at least, not that I could find. Anyway, it was a minor point - removed.

Reply:
Yeah not sure why the reason for using that format wasn't documented - that's my bad. I have a recollection of, at the time, it being less common to have ADRs across the programme so it felt more free form and there was some contention around "status" given it's pretty much a given they're accepted to be merged.